sqs/src/main/scala/akka/stream/alpakka/sqs/SqsAckFlowStage.scala

```diff
@@ -39,8 +35,8 @@ private[sqs] final class SqsAckFlowStage(queueUrl: String, sqsClient: AmazonSQSA
           checkForCompletion()
       }
 
-      private def handleSend(result: SendMessageResult): Unit = {
-        log.debug(s"Sent message {}", result.getMessageId)
+      private def handleChangeVisibility(result: ChangeMessageVisibilityResult): Unit = {
+        log.debug(s"Set visibility")
```

Comment:
I think it would be good to still have some information logged from the `result`.